corehq/apps/users/models.py

```diff
@@ -171,6 +171,7 @@ class Permissions(DocumentSchema):
     access_web_apps = BooleanProperty(default=False)
 
     edit_reports = BooleanProperty(default=False)
+    email_reports = BooleanProperty(default=True)
```

Comment:
Curious if you considered other names. I think of emailing as secondary to downloading, since emailing a report implies you can download it.

Reply:
gotcha I was debating that-- I'll change to `download_reports`

Reply:
Sorry for throwing in another requirements question, but is there a reason is true by default? Almost all permissions default to false.

Reply:
my understanding was that if we set it to `False`, then everyone that has these abilities to email/download previously would no longer be able to until that was changed

Reply:
Yeah, but that's what the migration is for: you can default it to false, but also set it to true for all existing roles, so that no one who currently has access will lose it.

Reply:
But that's a good thing to bring up not that the feature flag isn't there-- is that still what we want/ever wanted? Or is it more dependent on other permissions that is it should be set to `True` and `False` depending on how other permissions like `edit_reports` is set?

Reply:
> Yeah, but that's what the migration is for: you can default it to false, but also set it to true for all existing roles, so that no one who currently has access will lose it.

ah I see

Reply:
It's knowing there won't be a feature flag that brought this to mind, yes. If it were flagged it'd be a little bit more okay to be inconsistent with the other permissions, but it not being flagged makes me more strongly feel it should default to false.

Reply:
Thanks for flagging @orangejenny-- I offlined and looks like it will be set such that:
all roles that currently have this ability will retain the ability (permission set to True). Then similarly for new roles, it will be dependent on whether they can view reports, if they can it is set to True, otherwise it will be False. I need to dig into the code more to confirm, but looks like that will be dependent on whether they can `view_reports` or if `view_report_list` is not empty